src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -53,7 +53,11 @@ private void initUserAnswers() {
         for (Enumeration en = queryPrompts.keys(); en.hasMoreElements(); ) {
             String promptId = (String)en.nextElement();
             QueryPrompt prompt = queryPrompts.get(promptId);
-            userAnswers.put(prompt.getKey(), "");
+            String defaultValue = "";
+            if(prompt.getDefaultValueExpr() != null) {
+                defaultValue = FunctionUtils.toString(prompt.getDefaultValueExpr().eval(evaluationContext));
```

Comment:
Since this might result in XPathExceptions, we should make sure top level calls to this path are catching the exception and showing the error to the user.

Reply:
Good catch! I added the throws to the method signature for clarity, and added a catch for this in the Android PR.